src/auro-checkbox.js

```diff
@@ -50,6 +50,15 @@ class AuroCheckbox extends LitElement {
     };
   }
 
+  // This custom event is only for the purpose of supporting IE
+  // .addEventListener('change', function() { })
+  change(event) {
+    this.value = event.target.value
+    const customEvent = new event.constructor(event.type, event)
+
+    this.dispatchEvent(customEvent)
```

Comment:
```suggestion
    this.dispatchEvent(customEvent);
```
Semicolon for consistency 😄